src/elements/content-sidebar/stories/ContentSidebar.stories.js

```diff
@@ -1,25 +1,45 @@
 // @flow
+import * as React from 'react';
+import { http, HttpResponse } from 'msw';
 import ContentSidebar from '../ContentSidebar';
+import { mockFileRequest } from './__mocks__/ContentSidebarMocks';
+
+const defaultArgs = {
+    detailsSidebarProps: {
+        hasProperties: true,
+        hasNotices: true,
+        hasAccessStats: true,
+        hasClassification: true,
+        hasRetentionPolicy: true,
+    },
+    features: global.FEATURES,
+    fileId: global.FILE_ID,
+    hasActivityFeed: true,
+    hasMetadata: true,
+    hasSkills: true,
+    hasVersions: true,
+    token: global.TOKEN,
+};
 
 export const basic = {};
 
+export const withPanelPreservation = () => {
+    const features = { ...defaultArgs.features, panelSelectionPreservation: true };
+
+    return <ContentSidebar {...defaultArgs} features={features} />;
+};
+
```

Comment:
we should do something like this https://github.com/box/box-ui-elements/blob/master/src/elements/content-explorer/stories/ContentExplorer.stories.js instead

Reply:
Done